geniza/corpus/admin.py

```diff
@@ -1,8 +1,12 @@
 from django.contrib import admin
 
-from geniza.corpus.models import Library
+from geniza.corpus.models import Library, LanguageScript
 
 
 @admin.register(Library)
 class LibraryAdmin(admin.ModelAdmin):
     list_display = ('name', 'abbrev')
+
+@admin.register(LanguageScript)
+class LanguageScriptAdmin(admin.ModelAdmin):
+    list_display = ('language', 'script', 'display_name')
```

Comment:
would filter by script be useful?

Reply:
Stephanie was going to go through the issues and lay out all the specific filter and search she wanted for each, but didn't mention it for language. I don't think there are enough where that's immediately necessary. But give me a 🚀 and I'll throw it in!